content/ai/concepts/neural-networks/terms/convolutional-neural-networks/convolutional-neural-networks.md

```diff
@@ -0,0 +1,33 @@
+---
+Title: 'Convolutional Neural Networks'
+Description: 'Convolutional Neural Networks are a type of neural network that are primarily used for image classification.'
+Subjects:
+  - 'Computer Science'
+Subjects:
+  - 'Deep Learning'
+  - 'Computer Science'
+Tags:
+  - 'AI'
+  - 'Neural Networks'
+  - 'CNN'
+CatalogContent:
+  - 'machine-learning'
+  - 'paths/machine-learning-engineer'
+---
+
+# What is a CNN?
```

Comment:
```suggestion
```
Remove the first header, we don't have headers before the introductory paragraph.